metanorm/normalizers/netcdf_cf.py

```
@@ -0,0 +1,30 @@
+"""Normalizer for the CF Standard convention"""
+
+import logging
+
+import pythesint as pti
+import metanorm.utils as utils
+from .base import BaseMetadataNormalizer
+
+LOGGER = logging.getLogger(__name__)
+LOGGER.addHandler(logging.NullHandler())
+
+
+class NETCDFCFMetadataNormalizer(BaseMetadataNormalizer):
+    """ Search and find the corresponding the GeoSPaaS Datasetparameters using
+    raw_dataset_parameters attribute """
+
+    def get_dataset_parameters(self, raw_attributes):
+        """ Get the dataset's parameters """
+        if set(['raw_dataset_parameters']).issubset(raw_attributes.keys()):
+            standardized_dataset_parameters = list()
+            for raw_parameter_name in raw_attributes['raw_dataset_parameters']:
+                try:
+                    result = utils.two_list_search(raw_parameter_name)
+                except:
```

This is almost never a good idea: you will catch all exceptions, even unforeseen ones.
Please catch `IndexError`, which is the exception raise by pti methods when they don't find anything.

---

Done in the next commit.